src/applications/disability-benefits/all-claims/tests/pages/hospitalizationhistory.unit.spec.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
```

Comment:
can we camelCase this file name? `hospitalizationHistory`

Reply:
changed!